Gyanthakur/Dev_Profiles
# feat: add contact page for user support

## Description
- Add one contact button in header and footer
- Add new page for contact user

Fixes: #218 

## Type of Change


## How to Test
- First @Gyanthakur you need to open account in [formspree](https://formspree.io/)
- Then see page like below and copy the form endpoint
![image](https://github.com/user-attachments/assets/4de10ac0-0633-47a3-82ed-50e84b590e9d)
- Then go to code in contact-us.html file and paste that url into the action of from
![image](https://github.com/user-attachments/assets/3ef2b76d-5615-4602-80fb-7ad5649bc598)

# After Demo
[Screencast from 2024-10-10 12-06-38.webm](https://github.com/user-attachments/assets/33b3b8c7-1d92-4ab6-8807-08c76ad7e9f7)


## Checklist


## Additional Notes

Please add any other information that is relevant to this pull request, including potential risks, alternative solutions considered, or future improvements.
